qa/os/src/test/java/org/elasticsearch/packaging/test/KeystoreManagementTests.java

```diff
@@ -263,45 +238,43 @@ public void test51WrongKeystorePasswordFromFile() throws Exception {
      * and provide a password via an environment variable.
      */
     @AwaitsFix(bugUrl = "Keystore fails to save with resource busy")
```

Comment:
Is there an open issue for this?

Reply:
yes, it's https://github.com/elastic/elasticsearch/pull/76124. The AwaitsFix preceded the PR and I failed to update this